### PR TITLE
Search bar PR

### LINKED ISSUE #12: Search
Set up the searching task.
It should search through users, podcasts and series.
In addition, it should include categories and tags.
Creating the search bar.

### PR DESCRIPTION
**Pull request by ItzhakiGuy**
This PR is following to issue #12,
Creating a search bar in the HTML template of logged user,
Creating the logic of the search bar (search app - view),
Creating the search results view in HTML file.

### CODE REVIEW

**Review by 11galpid — CHANGES_REQUESTED** [review]
Works great, here's a screenshot guys:
image

**Review by 11galpid — APPROVED** [review]

File: sellotape/sellotape_dj/main_app/templates/sellotape/search.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{% extends "user.html" %}
```
Comment (alonweissfeld) [inline sellotape/sellotape_dj/main_app/templates/sellotape/search.html:1]: Isn't this should be on the home page? 🤔

File: sellotape/sellotape_dj/search/views.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        if stream_results is not None or profile_results is not None:
+            qs = sorted(queryset_chain, key=lambda instance: instance.pk, reverse=True)
+
+    count = length_streams + length_profiles
+    context = {
+        'qs': qs,
+        'query': query,
+        'count': count,
+    }
+    print(context)
```
Comment (alonweissfeld) [inline sellotape/sellotape_dj/search/views.py]: Please remove this or replace with informative logging if we want to log it..

File: sellotape/sellotape_dj/sellotape_dj/settings.py [hunk 3]
```diff
@@ -44,6 +44,7 @@
     'main_app.apps.MainAppConfig',
     'chat',
     'channels',
+    "search.apps.SearchConfig",
```
Comment (alonweissfeld) [inline sellotape/sellotape_dj/sellotape_dj/settings.py]: Try to be consistent, use single quote here :)

**Review by alonweissfeld — APPROVED** [review]
LGTM but see the comments 🐙

File: sellotape/sellotape_dj/main_app/templates/sellotape/search.html (hunk 1 above)
Comment (ItzhakiGuy) [reply sellotape/sellotape_dj/main_app/templates/sellotape/search.html:1]: This is the HTML that represents the results page, the form of the search bar is located in the navigation HTML file, as it should be located there while using the app.

**Comment by ItzhakiGuy** [comment]
Hi, I have uploaded a final version of the search bar, including comments (on search view) and changes according to your comments ;-)
The only thing left is rebasing the git commits and then I'll merge the PR.
@11galpid @alonweissfeld

**Pull request merged**